src/modules/oAuth/strategies/google.strategy.ts

```diff
@@ -0,0 +1,32 @@
+import { PassportStrategy } from '@nestjs/passport';
+import { Strategy, VerifyCallback } from 'passport-google-oauth20';
+import { Injectable } from '@nestjs/common';
+
+@Injectable()
+export class GoogleStrategy extends PassportStrategy(Strategy, 'google') {
+  constructor() {
```

Comment:
I think it is possible to inject config service here like this: 
```js
constructor(private configService: ConfigService) {}
```

and you can use it for getting env variables.

Reply:
used it